src/utils.php

```diff
@@ -131,3 +132,19 @@ function stripStringOverlap(string $a, string $b): string
     }
     return $b;
 }
+
+/**
+ * Use for sorting an array of URIs by number of segments
+ * in ascending order.
+ *
+ * Example:
+ *     usort($uriList, 'LanguageServer\sortByLeastSlashes');
+ *
+ * @param $a string
+ * @param $b string
+ * @return integer
+ */
+function sortByLeastSlashes($a, $b)
```

Comment:
make this function take the array and call `usort` on it
rename to `sortUrisLevelOrder`, it makes it more clear what it does